Vagrantfile

```diff
@@ -26,10 +28,9 @@ Vagrant.configure("2") do |config|
     sudo apt-get install -y opam m4 libxen-dev git
     opam init -a --compiler=4.02.3 -y
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
-    opam install -y merlin ocp-indent utop lwt_react depext camlp4
+    opam install -y merlin ocp-indent ocp-index ocp-browser utop lwt_react depext camlp4
     sudo opam depext -y xapi
-    opam pin add xenctrl 0.9.32
-    opam install ezxenstore
+    opam pin add lwt 2.7.1
```

Comment:
Differently from what I was hoping with https://github.com/xapi-project/xs-opam/issues/77, the pin is necessary. The presence of old version of our libraries makes the solver always find a configuration without `lwt` constraints, so if something is installing `lwt 3.0.0` (e.g. `lwt_react` installed above) the repository becomes unusable. The annoying alternative is to install the other packages after `xapi` dependencies, but it's too weak as a solution. I think it's better to use the pin as done here.